apple/internal/resource_rules/apple_core_ml_library.bzl

```diff
@@ -114,7 +114,7 @@ apple_core_ml_library = rule(
             allow_single_file = ["mlmodel"],
             mandatory = True,
             doc = """
-Label to a single `.mlmodel` file from which to generate sources and compile into mlmodelc files.
+Label to a single `.mlmodel` or `.mlpackage` file from which to generate sources and compile into mlmodelc files.
```

Comment:
could you split out the mlpackage changes into a separate PR and add a fixture like the one that currently exists for mlmodel?